packages/css/src/plugins/property-expander.tsx

```diff
@@ -0,0 +1,315 @@
+import { plugin, Declaration } from 'postcss';
```

Comment:
@madou: This file might get too big 😞 What should we do? Split in different files depending on property type? What you think 🤔? Tests are already touching 600 lines 😨

Reply:
Yeah we should